src/api/api.tsx

```diff
@@ -12,12 +13,18 @@ async function handleResponse<T>(response: Response): Promise<T> {
   return response.json();
 }
 
-export async function fetchLatestImages(limit = 6): Promise<UnsplashImage[]> {
-  const response = await fetch(`${BASE_URL}/photos?per_page=${limit}`, {
-    headers: {
-      Authorization: `Client-ID ${ACCESS_KEY}`,
-    },
-  });
+export async function fetchLatestImages(
+  page = 1,
+  limit = 6
+): Promise<UnsplashImage[]> {
+  const response = await fetch(
+    `${BASE_URL}/photos?page=${page}&per_page=${limit}`,
+    {
+      headers: {
+        Authorization: `Client-ID ${ACCESS_KEY}`,
+      },
```

Comment:
good option: create wrapper for all requests with headers and put headers only once. You will avoid headers duplication